### ISSUE TITLE
Site Editor: Navigation Drawer doesn't carry `:focus` between "layers."

### ISSUE THREAD

**Issue by shaunandrews** [WordPress/gutenberg#26070]
When using my keyboard to navigate the Navigation Drawer, I'm able to _dive into_ a section, but then there's no indicator of where my keyboard focus is when I do. In the following GIF, I use my keyboard to select "All" templates but then there is no focus state once the list of all templates is displayed:

<Img src="https://user-images.githubusercontent.com/191598/95894048-8be01880-0d56-11eb-96ef-4f325d39f77c.gif" width="200" />

I would expect my keyboard `:focus` to move to the top item in the list as I navigate through the layers.

**Comment by karmatosed** [comment]
Unless I am mistaken, this is no longer valid as this section has changed. I am, for now, going to close this. If it is at all valid, we can always reopen it, though, and absolutely should.
